mkdocs/docs/datastore/GET/episcanner.md

```diff
@@ -0,0 +1,72 @@
+## EpiScanner Data
+Here you get access to Real-time Epidemic Scanner data from the [Epi-Scanner project](https://info.dengue.mat.br/epi-scanner/), co-developed by the InfoDengue project. These data, described below, are based on the analyze of the expansion of dengue, zika and chikungunya in Brazil using up-to-date incidence data from Infodengue.
+
+## Parameters Table 
+### Input
+| Parameter name | Required | Type | Description |
+|--|--|--|--|
+| disease | yes | str | Specific disease. Options: dengue, zika, chik |
+| uf | yes | str _(UF)_ | Two letters brazilian's state abbreviation. E.g: SP |
+| year | no | int | Specific year. Default: current year |
+
+### Output
+| Parameter name | Type | Description |
+| -- | -- | -- |
+| disease | str | dengue, zika or chik
+| CID10 | str | Disease's CID
+| year | int | Data's year
+| geocode | int | City's geocode
+| muni_name | str | City's name
+| peak_week | float | #TODO
+| beta | float | #TODO
+| gamma | float | #TODO
+| R0 | float | #TODO
+| total_cases | int | Total cases in the last data update
+| alpha | float | #TODO
+| sum_res | float | #TODO
```

Comment:
Sum of residuals (see documentation).

Reply:
where should this [see documentation]() point to?